NEWS.md

```diff
@@ -1,3 +1,7 @@
+# naomi 2.5.4
```

Comment:
Typo here -- 2.5.4 -> 2.6.4.  (While your at it, add period at end of sentence: "Return input time series data in `area_sort_order` order**.**